integration/mediation-tests/tests-mediator-1/src/test/java/org/wso2/carbon/esb/mediator/test/cache/CacheTestCase.java

```diff
@@ -68,38 +68,33 @@ public void close() throws Exception {
     }
 
     @SetEnvironment(executionEnvironments = {ExecutionEnvironment.STANDALONE})
-    @Test(groups = {"wso2.esb"}, description = "Tests whether maxSize value evicts the new messages")
+    @Test(groups = {"wso2.esb"}, description = "Tests whether maxSize value evicts the older messages")
     public void testMaxSize() throws AxisFault, XPathExpressionException, InterruptedException {
+        String apiName = "maxSize";
         OMElement response;
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "ABCD");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "ABCD");
         String responseABCD1 = response.getFirstElement().toString();
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "ABC");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "ABC");
         String responseABC1 = response.getFirstElement().toString();
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "AB");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "AB");
         String responseAB1 = response.getFirstElement().toString();
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "AB");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "AB");
         String responseAB2 = response.getFirstElement().toString();
 
-        if (responseAB1.equalsIgnoreCase(responseAB2)) {
-            assertTrue(true, "The response AB is cached");
-        }
+        assertEquals(responseAB1, responseAB2, "The response for AB is cached");
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "ABC");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "ABC");
         String responseABC2 = response.getFirstElement().toString();
 
-        if (responseABC1.equalsIgnoreCase(responseABC2)) {
-            assertTrue(true, "The response ABC is cached");
-        }
+        assertEquals(responseABC1, responseABC2, "The response for ABC is cached");
 
-        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL("maxMessageOrSize"), "", "ABCD");
+        response = axis2Client.sendSimpleStockQuoteRequest(getApiInvocationURL(apiName), "", "ABCD");
         String responseABCD2 = response.getFirstElement().toString();
 
-        if (!responseABCD1.equalsIgnoreCase(responseABCD2)) {
-            assertTrue(true, "The response ABCD has been evicted");
-        }
+        assertNotEquals(responseABCD1, responseABCD2, "The response for ABCD has been evicted");
```

Comment:
Is the message printed when the assertion fails, correct?

Reply:
Fixed in #1167